web-server/Dockerfile

```
@@ -1 +1,21 @@
-# TODO: Build docker image with WebServer
\ No newline at end of file
+# Builder stage
+FROM python:3.8 AS builder
+COPY requirements.txt .
+
+# Install the required modules.
+RUN pip install -r requirements.txt
+
+# Publish stage
+FROM python:3.8 AS publish
```

```suggestion
FROM python:3.8-slim AS publish
```
Så sparer vi 290MB på det ferdige docker imaget 👍🏼 
https://hub.docker.com/_/python/?tab=tags&page=1&ordering=last_updated&name=3.8